templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="#">Home page</a></li>
+  <li><a href="#">Manufacturers</a></li>
+  <li><a href="#">Cars</a></li>
+  <li><a href="#">Drivers</a></li>

```

Comment:
The `href` attributes currently contain `#`, which is a placeholder. Replace these with the actual URL paths or use Django's `{% url %}` template tag to link to the correct views.